inverse/beamformer_lcmv.m

```diff
@@ -38,16 +38,21 @@
 %  'weightnorm'       = normalize the beamformer weights,                can be 'no', 'unitnoisegain' or 'nai'
```

Comment:
I guess the list needs to be extended by `'arraygain'`

Reply:
If you want to expose the option, that is - might be just for testing for now?

Reply:
Yes, indeed.